app/daemon.go

```diff
@@ -78,12 +86,18 @@ func startManager(c *cli.Context) error {
 	}
 	kubeconfigPath := c.String(FlagKubeConfig)
 
+	defaultSettingPath := c.String(FlagDefaultSettingPath)
+
 	if err := environmentCheck(); err != nil {
 		logrus.Errorf("Failed environment check, please make sure you " +
 			"have iscsiadm/open-iscsi installed on the host")
 		return fmt.Errorf("Environment check failed: %v", err)
 	}
 
+	if _, err := os.Stat(defaultSettingPath); err != nil {
```

Comment:
It should be allowed to be empty so we just use built-in internal defaults.

Reply:
I meant, if the path is empty, we should allow it.

But we indeed should error out if it's specified but cannot be found.